test/Confluent.Kafka.IntegrationTests/Tests/AdminClient_AlterConfigs.cs

```diff
@@ -34,6 +34,10 @@ public partial class Tests
         public void AdminClient_AlterConfigs(string bootstrapServers)
         {
             LogToFile("start AdminClient_AlterConfigs");
+            string onSemaphore = Environment.GetEnvironmentVariable("onSemaphore");
+            if(onSemaphore!=null){
+                return;
+            }
```

Comment:
We can move the `Environment.GetEnvironmentVariable` functionality into the `Tests.cs` file and create a static function there called `SkipSemaphoreFlakyTests`. Individual tests can use this function to decide whether to skip a test. 

Additionally, let's log using `LogToFile` whenever we skip a flaky test.